openadapt/strategies/visual.py

```diff
@@ -353,6 +356,33 @@ def find_similar_image_segmentation(
 
     return similar_segmentation, similar_segmentation_diff
 
+ 
+
+def combine_segmentations(
+    previous_segmentation: Segmentation,
+    new_descriptions: List[str],
+    new_masked_images: List[Image.Image],
+    new_masks: List[np.ndarray]
+) -> Segmentation:
+
+    combined_image = previous_segmentation.image
+    combined_masked_images = previous_segmentation.masked_images + new_masked_images
+    combined_descriptions = previous_segmentation.descriptions + new_descriptions
```

Comment:
I believe we need to filter out the `masked_images` and the `descriptions` that are not relevant for the new image, i.e. those that overlap with difference areas between the old and the new image.

Reply:
alrighty!